### PR TITLE
add button custom color

### PR DESCRIPTION
**Pull request by anistouri**

### CODE REVIEW

File: src/components/app-wrapper.js [hunk 1]
```diff
@@ -56,6 +56,9 @@ import { PARAM_THEME } from '../utils/config-params';
 const lightTheme = createTheme({
     palette: {
         mode: 'light',
+        customButton: {
+            main: '#000000DE',
```
Comment (sBouzols) [inline src/components/app-wrapper.js:60]: Maybe we should use values from the defaultTheme which is simpler : 
`theme.palette.text.primary` for example.
It's possible to define a custom theme color but is it useful ?
```
lightTheme = createTheme(theme, {
  palette: {
    customButtonColor: {
      main: theme.palette.text.primary,
    },
  },
});
```

**Review by sBouzols — CHANGES_REQUESTED** [review]

File: src/components/app-wrapper.js (hunk 1 above)
Comment (anistouri) [reply src/components/app-wrapper.js:60]: Done

**Review by sBouzols — CHANGES_REQUESTED** [review]

File: src/components/app-wrapper.js [hunk 2]
```diff
@@ -109,11 +117,27 @@ const darkTheme = createTheme({
     },
 });
 
+const customDarkTheme = createTheme(darkTheme, {
```
[the remark this answers is not in the document]
Comment (anistouri) [reply src/components/app-wrapper.js]: Done

File: src/components/app-wrapper.js [hunk 3]
```diff
@@ -84,6 +89,9 @@ const lightTheme = createTheme({
 const darkTheme = createTheme({
     palette: {
         mode: 'dark',
+        customButton: {
```
[the remark this answers is not in the document]
Comment (anistouri) [reply src/components/app-wrapper.js:88]: Done

File: src/components/app-wrapper.js [hunk 4]
```diff
@@ -56,6 +58,9 @@ import { PARAM_THEME } from '../utils/config-params';
 const lightTheme = createTheme({
     palette: {
         mode: 'light',
+        customButton: {
```
[the remark this answers is not in the document]
Comment (anistouri) [reply src/components/app-wrapper.js]: Done

File: src/components/app-wrapper.js [hunk 5]
```diff
@@ -109,11 +117,27 @@ const darkTheme = createTheme({
...
+            main: lightTheme.palette.text.primary,
+        },
+    },
+});
+
 const getMuiTheme = (theme) => {
     if (theme === LIGHT_THEME) {
-        return lightTheme;
+        return customLightTheme;
```
[the remark this answers is not in the document]
Comment (anistouri) [reply src/components/app-wrapper.js]: Done

**Review by sBouzols — APPROVED** [review]
Maybe change the `SubmitButton` interface définition in decs.d.ts in this project as well to respect the component in commons-ui
code Review OK

**Pull request merged**